### PR TITLE
Add a Reload All Instances API

### PR DESCRIPTION
**Pull request by Kneckter**
## Description
I made this API so we don't need to restart the whole container if we make edits to the instances in the DB and want RDM to use them. A simple GET will trigger it, if your account has admin perms.

http://127.0.0.1:9000/api/get_data?reload_instances=true

## Motivation and Context
This is being worked into an update to the SpawnpointClusterTool so it can update the DB and make RDM use the updates.

## How Has This Been Tested?
Compiled into my fork and used with the script/browser.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by 123FLO321 — DISMISSED** [review]

**Review by 123FLO321 — COMMENTED** [review]
I moved this function from `get_data` to `set_data` because it fits better there.
Please let me know if thats ok with you and if this sill works as expected.

**Comment by Kneckter** [comment]
> I moved this function from `get_data` to `set_data` because it fits better there.
> Please let me know if thats ok with you and if this sill works as expected.

I do get the idea but the reason I went with `get_data` was because `set_data` wouldn't accept the request. Give it a shot on your end, it's just a POST. The log doesn't trigger when I've tested it in that function.

File: Sources/RealDeviceMap/API/ApiRequestHandler.swift [hunk 1]
```diff
@@ -1830,6 +1831,14 @@ class ApiRequestHandler {
            } catch {
                response.respondWithError(status: .internalServerError)
            }
+	} else if reloadInstances && perms.contains(.admin) {
```
Comment (Fabio1988) [inline Sources/RealDeviceMap/API/ApiRequestHandler.swift:1834]: pls fix intent

**Comment by Fabio1988** [comment]
will be implemented in #268 thx @Kneckter 
it's working with that implementation if you send it with right CORS-Headers as POST or simple send GET.

**Pull request closed without merging**